src/plugins/legacy/polygonRoi/polygonEventFilter.cpp

```diff
@@ -266,30 +319,32 @@ QWidgetAction * polygonEventFilter::updateNameManager(medTagRoiManager* closestM
     QString style = "QLineEdit { color: rgb(%1, %2, %3); }";
     renameManager->setStyleSheet(style.arg(mycolor.red()).arg(mycolor.green()).arg(mycolor.blue()));
     connect(renameManager, &QLineEdit::editingFinished, [=](){
+       for (medTagRoiManager *manager : managers)
```

Comment:
there is an indentation problem on this paragraph

Reply:
maybe not because it's lamda function